### PR TITLE
[VEN-1248] Refactor: add path to swapPoolAssets

### PR DESCRIPTION
**Pull request by coreyar**
## Description

This PR adds an array of swap paths to the `swapPoolsAsset` method. This allows for multiple hops in the swap allowing better optimization for price

Resolves [VEN-1248]

### CODE REVIEW

File: contracts/RiskFund/RiskFund.sol [hunk 1]
```diff
@@ -148,14 +148,15 @@ contract RiskFund is
...
-        external
-        override
-        returns (uint256)
-    {
+    function swapPoolsAssets(
+        address[] calldata markets,
+        uint256[] calldata amountsOutMin,
+        address[][] calldata paths
+    ) external override returns (uint256) {
         _checkAccessAllowed("swapPoolsAssets(address[],uint256[])");
```
Comment (chechu) [inline contracts/RiskFund/RiskFund.sol:156]: The signature of the function has changed

File: tests/hardhat/Fork/RiskFund.ts [hunk 2]
```diff
@@ -623,7 +652,7 @@ describe("Risk Fund: Tests", function () {
         admin.address,
```
Comment (aditya-routv) [inline tests/hardhat/Fork/RiskFund.ts:652]: signature of the function changed

File: tests/hardhat/Fork/RiskFund.ts [hunk 3]
```diff
@@ -623,7 +652,7 @@ describe("Risk Fund: Tests", function () {
...
       );
       // Fails
-      await expect(riskFund.swapPoolsAssets([], [])).to.be.revertedWithCustomError(riskFund, "Unauthorized");
+      await expect(riskFund.swapPoolsAssets([], [], [])).to.be.revertedWithCustomError(riskFund, "Unauthorized");
 
       // Reset
       await accessControlManager.giveCallPermission(
```
Comment (aditya-routv) [inline tests/hardhat/Fork/RiskFund.ts:658]: likely above

Comment (coreyar) [reply tests/hardhat/Fork/RiskFund.ts:658]: fixed

**Review by aditya-routv — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**